### PR TITLE
Add info about user directory encryption option

### PR DESCRIPTION
**Pull request by pr0d1r2**

### CODE REVIEW

**Comment by drduh** [comment]
I'm not sure how this makes a single-user system more secure. It seems like an inconvenience. Perhaps it's worth expanding the 'create a second, non-administrator account' footnote into its own section first, then mention this there.

**Comment by pr0d1r2** [comment]
You are correct. This makes single-user system a little more secure (for sure adds extra unexpected layer for attacker). For multi-user systems it is quite nice as sometimes when there is need to share machine with family (which does not have so much paranoia) they tend to have simple passwords which can be used to decrypt hard drive.

**Comment by TraderStf** [comment]
@pr0d1r2 Hello, how do you make this? Thank you.

**Comment by pr0d1r2** [comment]
@TraderStf general idea is to use "Legacy FileVault" (present in 10.8) which is basicly sparsebundle keep in user directory and then mount on login. You can read details in: https://github.com/pr0d1r2/hackintosh_alpha/blob/master/setup_filevault_legacy.sh

**Comment by drduh** [comment]
I'm still not crazy for this; the guide really is geared towards single-user owners capable of being competent system administrators, and I don't want to complicate it unnecessarily. To be clear - I recognize the idea behind "security by isolation", I just don't think this is a particularly effective or worthwhile security measure. A recent vulnerability comes to mind which this wouldn't have mitigated, for example - https://truesecdev.wordpress.com/2015/04/09/hidden-backdoor-api-to-root-privileges-in-apple-os-x/

> I actually found a way to make it work for all users later, which means that the exploit is no longer limited to admin accounts only

What do others think about this PR?

**Comment by cmpis** [comment]
I think it makes sense in the situation he said which is a single computer used by many people. Which is currently alien to me, but certainly possible. Any harm in adding with the warning that it makes no difference if you have only a single person using that computer?

**Comment by drduh** [comment]
That sounds good, @cmpis 

@pr0d1r2, can you preface with something like, "If yours is a shared Mac, used by several individuals, consider ..."

**Comment by drduh** [comment]
Please just fix up formatting to match style of the rest of the doc and update wording as suggested above.

File: README.md [hunk 1]
```diff
@@ -262,6 +262,24 @@ Deploying FileVault 2](http://training.apple.com/pdf/WP_FileVault2.pdf) [pdf]
...
 and paper [Lest We Remember: Cold Boot Attacks on Encryption Keys](https://www.usenix.org/legacy/event/sec08/tech/full_papers/halderman/halderman.pdf) [pdf]
 
+## Separate user[s]
+
+Create a second, non-administrator account for day-to-day tasks. This
+will increase security by not having access to sudo (in case of its
+beeing compromised as it was couple of times in past). What you can do
+even more is apply some form of "security by isolation" (concept used
+widely in [Qubes OS](https://www.qubes-os.org/)) and create separate
+account for sensitive tasks like banking.
```
Comment (akshatpradhan) [inline README.md]: That's such a compromise on usability though.

**Comment by drduh** [comment]
Going to close this out. If anyone is feeling adventurous enough to create a more in-depth section about secondary, tertiary, etc. user accounts, be my guest. Right now this PR is incomplete and I don't care for making secondary accounts personally.

**Pull request closed without merging**